src/components/WalletTabs.vue

```diff
@@ -65,18 +62,18 @@ const logout = async () => {
             >
           </div>
         </div>
-        <div class="hidden md:flex items-center flex-1 ml-auto justify-end">
+        <div v-if="selectedAddress && user.verifierId" class="hidden md:flex items-center flex-1 ml-auto justify-end">
           <LanguageSelector class="mr-2" /><AccountMenu :user="user"
             ><AccountMenuList :user="user" :selected-address="selectedAddress" @on-logout="logout"
           /></AccountMenu>
         </div>
-        <div class="ml-6 flex md:hidden items-center">
+        <div v-if="selectedAddress && user.verifierId" class="ml-6 flex md:hidden items-center">
           <AccountMenuMobile><AccountMenuList :user="user" :selected-address="selectedAddress" @on-logout="logout" /></AccountMenuMobile>
         </div>
       </div>
     </nav>
 
-    <div class="py-6">
+    <div class="py-2 flex-auto overflow-y-auto w-full content-dimension">
```

Comment:
remove `content-dimension` , it causes weird 90% scrollbar. Replace with `py-6 sm:py-2` or something similar to add padding when mobile nav is visible.

Reply:
![image](https://user-images.githubusercontent.com/94340415/156098458-0ad22170-69da-42ee-8d5c-4cc1e3b370dc.png)
Current view

Reply:
fixed